content/guides/core-concepts/test-isolation.md

```diff
@@ -0,0 +1,124 @@
+---
+title: 'Test Isolation'
+---
+
+<Alert type="info">
+
+## <Icon name="graduation-cap"></Icon> What you'll learn
+
+- What is test isolation
+- How it impacts E2E Testing vs Component Testing
+- Test isolation trade-offs
+
+</Alert>
+
+## What is Test Isolation?
+
+<Alert type="success">
+
+<Icon name="check-circle" color="green"></Icon> **Best Practice:** Tests should
+always be able to be run independently from one another **and still pass**.
+
+</Alert>
+
+As stated in our mission, we hold ourselves accountable to champion a testing
+process that actually works, and have built Cypress to guide developers towards
+writing independent tests from the start.
+
+We do this by cleaning up state _before_ each test to ensure that the operation
```

Comment:
```suggestion
We do this by cleaning up browser state _before_ each test to ensure that the operation
```

Reply:
i know we clean up other state too, but it seems like browser state is the relevant state to this conversation 🤷‍♂️

Reply:
eh, i think this is fine in the context of the whole article.